MSAL/src/native_auth/public/MSALNativeAuthPublicClientApplication+Internal.swift

```diff
@@ -119,4 +119,19 @@ extension MSALNativeAuthPublicClientApplication {
         internalChallengeTypes.append(.redirect)
         return internalChallengeTypes
     }
+    
+    func getChallengeTypesFromInternalChallengeTypes(_ internalChallengeTypes: [MSALNativeAuthInternalChallengeType]) -> MSALNativeAuthChallengeTypes {
```

Comment:
nit: the function above `getInternalChallengeTypes()` is static. Can you make this static too?

Also there are two warnings from swiftlint.